lib/membrane/video_compositor/wgpu/wgpu_adapter.ex

```diff
@@ -1,165 +1,79 @@
 defmodule Membrane.VideoCompositor.WgpuAdapter do
   @moduledoc false
 
-  alias Membrane.VideoCompositor.RustStructs
-  alias Membrane.VideoCompositor.VideoTransformations
+  alias Membrane.VideoCompositor.Object.Layout
+  alias Membrane.VideoCompositor.Transformation
```

Comment:
the problem is that `Layout` is on one hand on the same abstraction layer as `Texture` (it's a node in a scene graph), but on the other on the same abstraction level as `Transformation` (is the set of parameters for a layout implementation and its module implements the behaviour connecting it to a `Layout` trait impl). If we move it like that we will have the same lack of consistency with `Texture` (why is one module that represents a scene object outside of the objects directory?)

Reply:
I don't really know how to tackle this, since splitting the current `Layout` into separate modules so that one is a scene node and the other contains the implementation seems unnecessary and just not very good to me

Reply:
What do you think about:
- Scene: `Membrane.VideoCompositor.Scene`
- Object: `Membrane.VideoCompositor.Scene.Object`
- Layout: `Membrane.VideoCompositor.Scene.Layout`
- Texture: `Membrane.VideoCompositor.Scene.Texture`
- Transformation: `Membrane.VideoCompositor.Scene.Transformation`

I know that it would make more sense to have:
- Scene: `Membrane.VideoCompositor.Scene`
- Object: `Membrane.VideoCompositor.Scene.Object`
- Layout: `Membrane.VideoCompositor.Scene.Object.Layout`
- Texture: `Membrane.VideoCompositor.Scene.Object.Texture`
- Transformation: `Membrane.VideoCompositor.Scene.Object.TextureTransformation`

but this level of module nesting is horrible to work with.

Reply:
seems ok, do you want me to change it in this PR or in a big renaming and restructuring PR that i'm planning to do soon?

Reply:
I would change it in this one, since after removing old VC stuff done here, the new project structure makes more sense.